Search/Header.swift

```diff
@@ -1,16 +1,16 @@
 //
 //  Header.swift
-//  Search
```

Comment:
Hmm, should probably put the module name in to the templates. I'll do that this morning.